artscii/src/services/stableDiffusionService.js

```diff
@@ -2,6 +2,11 @@ import axios from "axios";
 import home_gif from '../assets/home.gif'
 let Buffer = require('buffer').Buffer
 
+// default width and height of the image to keep API billing cost in check
+const DEFAULT_IMAGE_WIDTH = 512
+const DEFAULT_IMAGE_HEIGHT = 512
+const DEFAULT_UPSCALE = 2
```

Comment:
I agree with what @elizasohn said, we should set this to 1 or just let it default. I think when we set it to 2 we add cost going from 512 to 1024 resolution